owasp-suppressions.xml

```diff
@@ -115,4 +115,13 @@
     <packageUrl regex="true">^pkg:maven/io\.grpc/grpc\-.*@.*$</packageUrl>
     <cve>CVE-2023-44487</cve>
   </suppress>
+  <suppress until="2024-03-30Z">
+    <notes><![CDATA[
+   The fix might be available in 1.26.0, we will upgrade to it when its available
```

Comment:
The latest version is 1.25.0, and it also has the same vuln. So, for now, suppressing it.